app/models/work_version.rb

```diff
@@ -41,6 +41,12 @@ class WorkVersion < ApplicationRecord
     world: 'world'
   }
 
+  enum upload_type: {
+    browser: 'browser',
+    globus: 'globus',
+    hfszip: 'hfszip'
+  }
+
```

Comment:
not sure if we can come up with better names for these options (not shown to user, but stored in the database)